test/iron-image.html

```diff
@@ -207,6 +207,33 @@
 
             image.src = randomImageUrl();
           });
+
+          // Tests for PolymerElements/iron-image#115
+          test('image is not loaded when prevent-load is set', function() {
+            var attemptedLoadSpy = sinon.spy(image, '_setLoading');
```

Comment:
I think it would be better to check the external state of the image: `loading` / `loaded` / `error` and use the associated `*-changed` events to wait for changes. Some of the other tests earlier in this file are good examples.

However, if you're avoiding reading from `loading` because you don't feel confident that it accurately reflects the real state of the image, then we should add tests specifically for that. This would be nice but I don't think it's necessary to merge this PR.